kotsadm/web/src/components/AirgapUploadProgress.jsx

```diff
@@ -214,7 +214,7 @@ class AirgapUploadProgress extends React.Component {
               <div className="flex alignItems--center u-marginTop--20">
                 <span className="u-fontWeight--bold u-fontSize--normal u-color--tundora u-marginRight--10">{percentage}</span>
                 {progressBar}
-                {uploadComplete ? <span className="u-marginLeft--10 icon checkmark-icon" /> : <span className="u-fontWeight--medium u-fontSize--normal u-color--tundora u-marginLeft--10">{formatByteSize(total)}</span>}
+                {uploadComplete ? <span className="u-marginLeft--10 icon checkmark-icon" /> : <span className="u-fontWeight--medium u-fontSize--normal u-color--tundora u-marginLeft--10">{formatByteSize(progress)}</span>}{/* TODO: fix this */}
```

Comment:
What does `/*TODO: fix this */` mean?

Reply:
ah nice, good thing i added this, totally forgot about it. this used to show the total size of the bundle i think